### PR TITLE
Add LDAP service user info

### PR DESCRIPTION
**Pull request by mosabua**
As requested by @tnatssb

### CODE REVIEW

File: docs/src/main/sphinx/security/ldap.rst [hunk 1]
```diff
@@ -182,7 +184,8 @@ Property                                                Description
 ======================================================= ======================================================
 ``ldap.bind-dn``                                        Bind distinguished name used by Trino when issuing
                                                         group membership queries.
-                                                        Example: ``CN=admin,OU=CITY_OU,OU=STATE_OU,DC=domain``
+                                                        Example values: ``CN=admin,OU=CITY_OU,OU=STATE_OU,DC=domain
```
Comment (hashhar) [inline docs/src/main/sphinx/security/ldap.rst:187]:
```suggestion
                                                        Example values: ``CN=admin,OU=CITY_OU,OU=STATE_OU,DC=domain``
```

Comment (mosabua) [reply docs/src/main/sphinx/security/ldap.rst]: Fixed

**Review by electrum — APPROVED** [review]

File: docs/src/main/sphinx/security/ldap.rst [hunk 2]
```diff
@@ -171,6 +171,8 @@ Authorization using Trino LDAP service user
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Trino server can use dedicated LDAP service user for doing user group membership queries.
+You can use this configuration to support multiple domains.
```
Comment (electrum) [inline docs/src/main/sphinx/security/ldap.rst:174]: How?

**Comment by mosabua** [comment]
Reminder for myself - https://github.com/trinodb/trino/pull/8134

**Comment by mosabua** [comment]
Will reopen

**Comment by mosabua** [comment]
Will reopen when I had time to improve this.

**Pull request closed without merging**